src/border.jl

```diff
@@ -200,12 +210,12 @@ should be replicated by three and four pixels.
 """
 Pad(lo::Dims, hi::Dims) = Pad(:replicate, lo, hi)
 Pad(style::Symbol, lo::Tuple{}, hi::Tuple{}) = Pad{0}(style, lo, hi)
-Pad(style::Symbol, lo::Dims{N}, hi::Tuple{}) where {N} = Pad(style, lo, ntuple(d->0,Val(N)))
-Pad(style::Symbol, lo::Tuple{}, hi::Dims{N}) where {N} = Pad(style, ntuple(d->0,Val(N)), hi)
+Pad(style::Symbol, lo::Dims{N}, hi::Tuple{}) where {N} = Pad(style, lo, ntuple(d -> 0, Val(N)))
+Pad(style::Symbol, lo::Tuple{}, hi::Dims{N}) where {N} = Pad(style, ntuple(d -> 0, Val(N)), hi)
```

Comment:
Thanks for the commit, I really appreciate that. The following is my personal view, please fell free to correct me.

30 file changes in one commit might require more careful attention during code review. Additionally, some of these **changes appear to be related to formatting strings**. Perhaps we can split this into two separate changes: one for adding a new feature and another for changing the format.

Nonetheless, the file appears to be generally in good condition.